unit/unit_writer/unit_writer.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2018-2022 Jolla Ltd.
- * Copyright (C) 2018-2022 Slava Monich <slava.monich@jolla.com>
+ * Copyright (C) 2018-2023 Slava Monich <slava@monich.com>
```

Comment:
```suggestion
 * Copyright (C) 2023 Slava Monich <slava@monich.com>
```
This should actually be like this.

Reply:
ok))